doc/classes/ProjectSettings.xml

```diff
@@ -3205,6 +3205,10 @@
 			[b]Note:[/b] This property is only read when the project starts. There is currently no way to change this value at run-time.
 			[b]Note:[/b] Some platforms may restrict the actual value.
 		</member>
+		<member name="rendering/rendering_device/vsync/wait_for_present" type="bool" setter="" getter="" default="true">
+			Instructs Godot to use Waitable Swapchains when supported. Normally Godot allows the GPU to get [member rendering/rendering_device/vsync/frame_queue_size] ahead of the GPU. Assuming frame_queue_size = 2, Godot normally starts working on frame N+2 with the CPU when the GPU is done working on frame N. This setting makes Godot start frame N+2 when frame N is [i]done presenting[/i].
+			This gap between when the GPU finishes its work and when that work actually appears on screen can increase latency. [b]Enabling this setting thus reduces latency[/b] at the cost of some framerate (framerate may decrease because the CPU has to wait more time doing nothing).
```

Comment:
```suggestion
			Instructs Godot to use Waitable Swapchains when supported. Normally, Godot allows the GPU to get [member rendering/rendering_device/vsync/frame_queue_size] ahead of the GPU. Assuming [code]frame_queue_size = 2[/code], Godot normally starts working on frame [code]N + 2[/code] with the CPU when the GPU is done working on frame [code]N[/code]. This setting makes Godot start frame [code]N + 2[/code] when frame [code]N[/code] is [i]done presenting[/i].
			This gap between when the GPU finishes its work and when that work actually appears on screen can increase latency. [b]Enabling this setting thus reduces latency[/b] at the cost of some framerate (framerate may decrease because the CPU has to wait more time doing nothing).
```